pyshorteners/base.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, **kwargs):
         self.verify = getattr(self, "verify", True)
         self.proxies = getattr(self, "proxies", {})
         self.cert = getattr(self, "cert", None)
+        self.clean = getattr(self, "clean", True)
```

Comment:
this should a parameter in the function and not a class attribute. Can you move that to the function?

Reply:
Sorry for the delay. Is it okay now? @ellisonleao